pkg/opencontainer/specs.go

```diff
@@ -0,0 +1,437 @@
+package opencontainer
+
+import (
+	"os"
+)
+
+// These struct definitions are copied verbatim from "github.com/opencontainers/specs".
+// The original source package includes Linux-specific structures configured to be
+// compiled only on Linux systems, making it unnecessarily difficult to develop P2 from
+// other environments.
```

Comment:
Any structure with "Linux" in the name.  Seriously, this is a giant concatenation of 4 files with their headers aggregated.  Upgrading this file isn't going to be a problem.